admin/class-yoast-form.php

```diff
@@ -653,4 +669,34 @@ public function show_hide_switch( $var, $label, $inverse_keys = false, $help = '
 
 		$this->toggle_switch( $var, $show_hide_switch, $label, $help );
 	}
+
+	/**
+	 * Checks whether a given control should be disabled.
+	 *
+	 * @param string $var The variable within the option to check whether its control should be disabled.
+	 *
+	 * @return bool True if control should be disabled, false otherwise.
+	 */
+	protected function is_control_disabled( $var ) {
+		return $this->option_instance->is_disabled( $var );
+	}
+
+	/**
+	 * Prints a explanation note if a given control is disabled.
+	 *
+	 * @param string $var The variable within the option to print a disabled note for.
+	 *
+	 * @return void
+	 */
+	protected function print_disabled_note( $var ) {
+		if ( ! $this->is_control_disabled( $var ) ) {
+			return;
+		}
+
+		?>
+		<div class="disabled-note">
```

Comment:
I would choose for a situation where the output is returned instead of echoed directly